solidity/contracts/bridge/Bridge.sol

```diff
@@ -1302,7 +1302,8 @@ contract Bridge is
     /// @dev Requirements:
     ///      - Redemption dust threshold must be greater than moving funds dust
     ///        threshold,
-    ///      - Redemption treasury fee divisor must be greater than zero,
+    ///      - Redemption dust threshold must be greater than the redemption TX
+    ///        max fee,
     ///      - Redemption transaction max fee must be greater than zero,
```

Comment:
We are missing one requirement in the description of `updateRedemptionParameters` functions:
```
- redemption transaction max total fee must be greater or equal to redemption transaction max fee
```

Reply:
Eagle eye! 0b16550757d9b19d018e68d74864949a593b7e17